GUI/src/cz/fidentis/gui/PairComparisonPanel.java

```diff
@@ -733,9 +743,14 @@ private void secondaryColorPanelMouseClicked(java.awt.event.MouseEvent evt) {//G
         colorDialog.setVisible(true);
     }//GEN-LAST:event_secondaryColorPanelMouseClicked
 
-    private void jSlider3StateChanged(javax.swing.event.ChangeEvent evt) {//GEN-FIRST:event_jSlider3StateChanged
-        setColors();
-    }//GEN-LAST:event_jSlider3StateChanged
+    private void transparencySliderStateChanged(javax.swing.event.ChangeEvent evt) {//GEN-FIRST:event_transparencySliderStateChanged
+        if (!transpTresholdValueChanged) {
+            transpTresholdValueChanged = true;
+            transpSpinner.setValue((int)transparencySlider.getValue());
```

Comment:
vid. review, je potrebne nastavovat hodnotu transparency v datovom module, sa mi ale vidi, ze momentalny kod v developmente vyzera trocha inak, ako to co bolo prepisane podla diffu